### DISCUSSION TITLE
[Parte 2 pregunta 5 - GIVE-PUFFLES] Puffles pingüino 2

### DISCUSSION THREAD

**Discussion by MystEdu** [IIC2133-PUC/2024-2#49] [Tarea 0] [answered]
Hola buenas,
El pingüino que recibe (pingüino 2) ¿siempre tiene al menos 1 puffle en su inventario? o ¿puede recibir puffles teniendo 0 en su inventario?
Pregunto para ver si tengo que implementar ese caso específico en mi código.

**Comment by tomastrivino** [comment, marked as answer]
Te recomiendo que consideres ambos casos.
